testing/test_warnings.py

```diff
@@ -19,7 +19,7 @@ def pyfile_with_warnings(pytester: Pytester, request: FixtureRequest) -> str:
     """Create a test file which calls a function in a module which generates warnings."""
     pytester.syspathinsert()
     test_name = request.function.__name__
-    module_name = test_name.lstrip("test_") + "_module"
+    module_name = test_name.lstrip("tes_") + "_module"
```

Comment:
I think the `lstrip` here is a guniuine mistake, it probably meant to be `removeprefix("test_")`. However `removeprefix` was only added in Python 3.9, so can do `test_name[len("test_"):]` or such.

Reply:
4a3d751c6 (Assuming that "test_" will always be here, seems reasonable.)